sdk/cognitiveservices/InkRecognizer/tests/InkRecognizerTest.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
This should be `    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>`